roles/variables/defaults/main.yml

```diff
@@ -215,6 +215,21 @@ certificate_authority_expiration_days: 365
 ### Boolean to enable mTLS Authentication on all components. Configures all components to use mTLS for authentication into Kafka
 ssl_mutual_auth_enabled: false
 
+### mTLS server's config to enforce ssl client authentication. Options are none, requested, required
+ssl_client_authentication: >-
+  {%- if ssl_mutual_auth_enabled|bool -%}
+    {%- if deployment_strategy == 'parallel' -%}
+      required
+    {%- elif deployment_strategy in ['serial', 'rolling'] -%}
```

Comment:
If not defined in case of deployment strategy serial/rolling then we should fail the setup